<?xml version="1.0" encoding="UTF-8"?>
<launch>

    <arg name="name"/>
    <arg name="session"/>

    <!-- robot name -->
    <param name="robot_name" type="str" value="$(arg name)"/>

    <!-- current session tag -->
    <param name="session_tag" value="$(arg session)"/>

    <!-- perception subsystem -->
    <include file="perception.launch">
        <arg name="name" value="$(arg name)"/>
    </include>

</launch>
